Dark mode text needs higher contrast (#1987)
jazzband/django-debug-toolbar: docs/changes.rst

  ``FORCE_SCRIPT_NAME`` was set.
* Increase opacity of show Debug Toolbar handle to improve accessibility.
* Changed the ``RedirectsPanel`` to be async compatible.

> * Fixed the Dark mode text needs higher contrast.

Reviewer: ```suggestion
* Increased the contrast of text with dark mode enabled.
```
